templates/base.html

```diff
@@ -59,42 +67,49 @@
-                <span class="text">
-                    <i class="icon user outline"></i>
-                    {{ user.username }}
-                </span>
-                <i class="dropdown icon"></i>
-                <span class="menu">
-                {% if user.is_staff or user.is_superuser %}
-                    <div class="header">Django Admin</div>
-                    <a class="item" href="{% url 'admin:index' %}">Index</a>
-                    <a class="item" href="{% url 'admin:index' %}homework/">Homework</a>
-                    <a class="item" href="{% url 'admin:index' %}klasses/">Classes</a>
-                    <a class="item" href="{% url 'admin:index' %}profiles/">Profiles</a>
-                {% endif %}
-                    <div class="ui divider"></div>
-                    <div class="header">Chasuite</div>
-                        <a class="item" href="https://competitions.codalab.org/">Codalab</a>
-                        <a class="item" href="https://chagrade.lri.fr/">Chagrade</a>
-                        <a class="item" href="https://chalab.lri.fr/">Chalab</a>
-                        <a class="item" href="#">Chagle</a>
-                    <div class="ui divider"></div>
-                    <div class="header">My Account</div>
-                    <a class="item" href="/profiles/my_profile">
-                        <i class="icon user"></i>
-                        My profile
-                    </a>
-                    <a class="item" href="/accounts/logout/">
-                        <i class="icon sign out"></i>
-                        Logout
-                    </a>
+                    <span class="text">
+                        <i class="icon user outline"></i>
+                        {{ user.username }}
+                    </span>
+                    <i class="dropdown icon"></i>
+                    <span class="menu">
+                    {% if user.is_staff or user.is_superuser %}
+                        <div class="header">Django Admin</div>
+                        <a class="item" href="{% url 'admin:index' %}">Index</a>
+                        <a class="item" href="{% url 'admin:index' %}homework/">Homework</a>
+                        <a class="item" href="{% url 'admin:index' %}klasses/">Classes</a>
+                        <a class="item" href="{% url 'admin:index' %}profiles/">Profiles</a>
+                        <a class="item" href="{% url 'metrics:admin' %}">Metrics</a>
+                    {% endif %}
+                        <div class="ui divider"></div>
+                        <div class="header">Chasuite</div>
+                            <a class="item" href="https://competitions.codalab.org/">Codalab</a>
+                            <a class="item" href="https://chagrade.lri.fr/">Chagrade</a>
+                            <a class="item" href="https://chalab.lri.fr/">Chalab</a>
+                            <a class="item" href="#">Chagle</a>
```

Comment:
This should probably be Chahub I think... (Which we can link to! `https://chahub.org`)